README.md

```diff
@@ -12,15 +12,15 @@ A simple Elixir package for implementing [reCAPTCHA] in Elixir applications.
 
 [reCAPTCHA]: http://www.google.com/recaptcha
 
-## Migration from 1.x
+## Migration from 1.x to 2.x
```

Comment:
Should we have a short 4.x section?

```
## Migration from 3.x to 4.x

- Now requires elixir version 1.13 or later
- The recaptcha widget has been removed. If you need this we suggest you inline it in your project from the previous version of this library.
```

Reply:
Nice suggestion! I have pushed [this commit](https://github.com/remoteoss/recaptcha/pull/1/commits/ee1adb243dbb6940684808aac8e768c7159ea8e9) which adds that section, removes some of the badges as they were pointing to the original repo and updates the installation instructions, as our version of the library is not on hex.pm . Let me know what you think 🙂

Reply:
Excellent! ✨